data/data/azure/dns/variables.tf

```diff
@@ -59,3 +59,13 @@ variable "private" {
   type        = bool
   description = "This value determines if this is a private cluster or not."
 }
+
+variable "azure_supports_private_dns" {
```

Comment:
This could be used to distinguish between legacy and new private DNS in the previous file.